src/components/implementation/no_interface/cFE_booter/cFE_emu_support.c

```diff
@@ -0,0 +1,117 @@
+#include <stdio.h>
+
+#include <cos_component.h>
+#include <cos_debug.h>
+#include <cos_types.h>
+
```

Comment:
We follow the std => cos => interface => cFE => internal header convention. Otherwise it becomes very confusing where each header is coming from. Especially since we have 5 header sources. Is this something the cos style guide takes a strong stance on?